pointfog/src/layers/Scrollbar.js

```diff
@@ -33,14 +33,24 @@ export default class Scrollbar {
         this.width = 20;
     }
 
-    mouseClicked(): void {}
-    mousePressed(): void {}
+    mouseClicked(): void { }
+    mousePressed(): void {
+        if (this.p.mouseButton === this.p.RIGHT) {
+            if (this.p.mouseX < this.left + this.width && this.p.mouseY < this.top + this.height) {
+                this.im.setZ(Math.round(this.im.nSlices * (this.p.mouseY - this.top) / (this.height - this.top)));
+                return false;
+            }
+        }
+        return true;
+    }
 
     getEntities() {
+        let offset = 5;
```

Comment:
In breadcrumbs/../../Scrollbar.js you set the offset as a constant - fine either way, may be good to keep as much consistency in implementations where possible